flowtype/flow-typed
# Add a test runner

`flow-typed run-tests [opt-test-name-pattern]`

Running tests means the following: 

1. For each Package/Version/FlowVersion directory, find all versions of Flow that satisfy the specified Flow version
1. Copy the libdef + all test files into a directory with a .flowconfig specifying a "$ExpectError" suppression comment
1. Run `flow check` for each resolved version of Flow on the test directory. If there are any errors, assert them.

This also adds `run-tests` to the travis command
